src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -405,7 +415,13 @@ public sealed partial class ErrorWrapper
         public ErrorWrapper(object errorCode) { }
         public int ErrorCode { get { throw null; } }
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(5149), Inherited = false)]
+    public enum ExporterEventKind
```

Comment:
These exporter types shouldn't be exposed.

Reply:
Seems similar to the other APIs that were initially meant to be exposed as part of https://github.com/dotnet/corefx/issues/12162. cc @alexperovich, @danmosemsft 

Shows up in this diff:
https://github.com/dotnet/standard/blob/205332796a9a0f869ff698402c3496f14cc71f50/docs/comparisons/netstandard2.0_vs_net461/System.Runtime.InteropServices.md
And is in netfx:
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.exportereventkind?view=netframework-4.7.2